app/components/LifiWidgetPopover.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { Button, Popover, PopoverContent, PopoverTrigger } from '@swapr/ui';
+import { LifiWidget } from '@/app/components/LifiWidget';
+import { SDAI } from '@/constants';
+import { TokenLogo } from '@/app/components/TokenLogo';
+
+export const LifiWidgetPopover = () => {
+  return (
+    <Popover>
+      <PopoverTrigger asChild>
+        <Button variant="pastel" className="space-x-2 text-nowrap" onClick={() => {}}>
+          <p>Get {SDAI.symbol}</p>
+          <TokenLogo address={SDAI.address} size="xs" className="hidden md:block" />
+        </Button>
+      </PopoverTrigger>
+      <PopoverContent className="max-w-80 px-4 md:max-w-lg">
+        <LifiWidget />
```

Comment:
Should we delete this component and have the logic here? I think in this case there is not so much code, so it's good to have everything in 1 file.

Reply:
yeah, that's a good idea.